src/doc/trpl/concurrency.md

```diff
@@ -343,12 +343,12 @@ threads as a simple isolation mechanism:
 ```rust
 use std::thread;
 
-let result = thread::spawn(move || {
+let handle = thread::spawn(move || {
     panic!("oops!");
-}).join();
+});
 
-assert!(result.is_err());
+assert!(handle.join().is_err());
```

Comment:
I would actually prefer if we moved this `join()` out, instead: joining in an `assert!` feels very...strange.